pkg/hub/controllers/vpnkeyrotation/reconciler.go

```diff
@@ -366,10 +366,10 @@ func (r *Reconciler) syncCurrentRotationState(ctx context.Context,
 			return getErr
```

Comment:
@YachikaRalhan this method returns (bool, error) you are returning one value

Reply:
@rajendra-avesha that is under errors.Retry which has a callback function returning error only